content/efcore/getting-started/samples/loading-related-data-tutorial/then-include-method-chain/Program.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public class Program
+{
+    public static void Main()
+    {
+        using (var context = new LibraryContext())
+        {
+            var author = context.Authors
+                .Single(a => a.LastName == "Douglass")
+                .Include(a => a.Books)
+                    .ThenInclude(b => b.Editions)
+                        .ThenInclude(e => e.Publisher);
+            Console.WriteLine(String.Format("{0} : {1} - {2}", book.Title, book.Editions, edition.Publisher));
```

Comment:
This needs to be reworked, because the author is loaded, not the book, and both `Books` and `Editions` are `Lists`. Maybe we could just do `First` on the books, and then a `foreach` for the editions?